components/QuestionTypeCard/index.tsx

```diff
@@ -17,7 +17,7 @@ const QuestionTypeCard: React.FC<QuestionTypeCardProps> = ({ questionType, handl
 
   const json = JSON.parse(questionType.json);
   const questionTypeId = json.type;
-  const qType = JSON.stringify(questionTypeId)?.trim().replace(/^"|"$/g, '');//Since the question type was taken from the JSON string, it could have quotes or slashes that need to be removed.
+  let qType = JSON.stringify(questionTypeId)?.trim().replace(/^"|"$/g, '');//Since the question type was taken from the JSON string, it could have quotes or slashes that need to be removed.
```

Comment:
Just wondering the reason this changed to `let` instead of `const`? It doesn't look like the qType value changes elsewhere in the component.